### ISSUE TITLE
Verification setups

### ISSUE THREAD

**Issue by mithro** [timvideos/HDMI2USB-jahanzeb-firmware#26]
Please check that HDMI2USB works with the following set ups;
# Set Up 1
- Capture PC: Windows 7
- Input PC: Windows 7

_I only included this set up as I believe it is what you are testing with at the moment right?_
# Set Up 2
- Capture PC: Ubuntu Precise
- Input: Ubuntu Precise computer
# Set Up 3
- Capture PC: Ubuntu Precise
- Input: Windows 7 computer
# Set Up 4
- Capture PC: Ubuntu Precise
- Input: Mac OS X computer
# Set Up 5
- Capture PC: Ubuntu Precise
- Input: Raspberry PI

_I can send you a Raspberry Pi if you don't have one._
# Set Up 6
- Capture PC: Ubuntu Precise
- Input: PS3 or Xbox 360 or Nintendo Wii

_This is a low priority, but I would like to see what happens - I think HDCP is probably going to make it busted..._

**Comment by micolous** [comment]
I own an Xbox 360 so can test out things with that.

Input appears to work with the Xbox 360 but there are issues getting the resolution set properly (#30).  Once this is done it works fine at 1024x768@60Hz.  I am able to do this with no special tricks (provided that @jahanzeb's firmware isn't attempting HDCP magic.  I'm able to play games, and even video apps:

![p-07_12_2013-03_14_02](https://f.cloud.github.com/assets/246847/783497/b2b9152c-ea51-11e2-9aff-d933ec8c0a36.jpg)

Nintendo Wii only has Component and Composite outputs.  The Wii U has HDMI output available but I don't know if it requires HDCP.

Playstation 3 I'm fairly sure is always-on HDCP.

I'll attempt to get in touch with someone to test it out with those other consoles.

**Comment by micolous** [comment]
I attempted to capture from a Playstation 3 on the weekend, but was unsuccessful.  I forgot to log the LED output when I was trying.  I'm fairly sure this is to do with HDCP.  So I think that testing against the PS3 is a dead-end.

I was not able to borrow a Wii U over the weekend in order to test this.  I may be able to at another time.

**Comment by mithro** [comment]
This issue was moved to timvideos/HDMI2USB-misoc-firmware#163
